kubectl-pg/cmd/addDb.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"log"
+)
+
+// addDbCmd represents the addDb command
+var addDbCmd = &cobra.Command{
+	Use:   "add-db",
+	Short: "Adds a DB and its owner to a Postgres cluster. The owner role is created if it does not already exist",
+	Long:  `Adds a DB and its owner to the cluster owner needs to be added with -o flag.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 {
+			dbName := args[0]
+			dbOwner, _ := cmd.Flags().GetString("owner")
+			clusterName, _ := cmd.Flags().GetString("clusterName")
+			addDb(dbName, dbOwner, clusterName)
+		} else {
+			fmt.Println("database name can't be empty.")
+		}
+
+	},
+	Example: "kubectl pg add-db [DB-NAME] -o [OWNER-NAME] -c [CLUSTER-NAME]",
+}
+
+// add db and it's owner to the cluster
+func addDb(dbName string, dbOwner string, clusterName string) {
```

Comment:
Executing this command with 
`kubectl pg add-db test-db -o test-user -c some-test-db-name` 
results in the exception
```
panic: assignment to entry in nil map

goroutine 1 [running]:
kubectl-pg/cmd.addDb(0x7ffcc003722b, 0x7, 0x7ffcc0037236, 0x9, 0x7ffcc0037243, 0x13)
        /home/sdudoladov/go/src/github.com/zalando/postgres-operator/kubectl-pg/cmd/addDb.go:58 +0x67c
kubectl-pg/cmd.glob..func1(0x2211120, 0xc0002b40a0, 0x1, 0x5)
        /home/sdudoladov/go/src/github.com/zalando/postgres-operator/kubectl-pg/cmd/addDb.go:36 +0x107
github.com/spf13/cobra.(*Command).execute(0x2211120, 0xc0002b4050, 0x5, 0x5, 0x2211120, 0xc0002b4050)
        /home/sdudoladov/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x2212520, 0xc000123f68, 0x1203b3e, 0x2212520)
        /home/sdudoladov/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/sdudoladov/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
kubectl-pg/cmd.Execute()
        /home/sdudoladov/go/src/github.com/zalando/postgres-operator/kubectl-pg/cmd/root.go:33 +0x32
main.main()
        /home/sdudoladov/go/src/github.com/zalando/postgres-operator/kubectl-pg/main.go:22 +0x20
```

Please fix the error.

UPD check  the case where the manifest originally  has no databases defined (i.e. [this section](https://github.com/zalando/postgres-operator/blob/master/manifests/minimal-postgres-manifest.yaml#L16) is missing)

Reply:
fixed this.